src/App.js

```diff
@@ -14,11 +15,125 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goodsDatas: goodsFromServer,
+    start: true,
+    isReversed: false,
+    sortBy: 'alphabet',
+  }
+
+  start = () => {
+    this.setState({ start: false });
+  };
+
+  reverse = () => {
+    this.setState(state => ({
+      isReversed: !state.isReversed,
+    }));
+  };
+
+  sortAlphabetically = () => {
+    this.setState({
+      sortBy: 'alphabet',
+      isReversed: false,
+    });
+  };
+
+  reset = () => {
+    this.setState({
+      sortBy: '',
+      isReversed: false,
+    });
+  };
```

Comment:
```suggestion
  reset = () => {
    this.setState({
      goodsDatas: goodsFromServer
      isReversed: false,
    });
  };
```

You could do it just like this. It would be more readable and understandable